docs/docusaurus/docs/core/run_validations/run_a_validation_definition.md

```diff
@@ -45,7 +45,7 @@ import PrereqValidationDefinition from '../_core_components/prerequisites/_valid
 
    Validation Results are automatically saved in your Data Context when a Validation Definition's `run()` method is called.  For convenience, the `run()` method also returns the Validation Results as an object you can review.
 
-   :::tip
+   :::tip Result verbosity
```

Comment:
note for reviewer: I added more specific titles for these tips since there are two tips so close to each other on this page.